### PR TITLE
Update .env.sample

### PR DESCRIPTION
**Pull request by bartoszWojciechO**

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -1,22 +1,31 @@
...
+# Domains for analytics.
 EVENTS_PROD_DOMAIN=http://example.com
 EVENTS_STAGING_DOMAIN=http://example.com
 EVENTS_SUBDOMAIN=subdomain
+# App features, possible values are:
+# * telio
+# * drop
+# * moose - internal builds only
+# * internal - internal builds only
 FEATURES="telio drop"
```
Comment (piotrjurkiewicz) [inline .env.sample:28]: I get errors when compiling with FEATURES in quotation marks. Marius also wrote about that in onboarding resources. So maybe they should be removed from the example file?

File: .env.sample [hunk 2]
```diff
@@ -1,22 +1,31 @@
...
 VALID_LOGIN_USERNAME=valid
-VALID_LOGIN_PASSWORD=*********
 DEFAULT_LOGIN_TOKEN=0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef
 EXPIRED_LOGIN_USERNAME=expired
-EXPIRED_LOGIN_PASSWORD=********
 DEFAULT_LOGIN_TOKEN=0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef
+# Credentials used for meshnet testing. QA peer and Default accounts connect with each other via meshnet.
 QA_PEER_USERNAME=qa
 QA_PEER_PASSWORD=********
 DEFAULT_LOGIN_TOKEN=0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef
```
Comment (piotrjurkiewicz) [inline .env.sample:14]: Should be QA_PEER_TOKEN

File: .env.sample [hunk 3]
```diff
@@ -1,22 +1,31 @@
...
+# Salt used for encrypting the configuration.
 SALT=f1nd1ngn3m0
+# Default login credentials, use in all of the python tests.
 DEFAULT_LOGIN_USERNAME=default
-DEFAULT_LOGIN_PASSWORD=********
 DEFAULT_LOGIN_TOKEN=0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef
+# Credentials, used for login testing.
 VALID_LOGIN_USERNAME=valid
-VALID_LOGIN_PASSWORD=*********
 DEFAULT_LOGIN_TOKEN=0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef
```
Comment (piotrjurkiewicz) [inline .env.sample:8]: Should be VALID_LOGIN_TOKEN

File: .env.sample [hunk 4]
```diff
@@ -1,22 +1,31 @@
...
 EXPIRED_LOGIN_USERNAME=expired
-EXPIRED_LOGIN_PASSWORD=********
 DEFAULT_LOGIN_TOKEN=0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef
```
Comment (piotrjurkiewicz) [inline .env.sample:10]: Should be EXPIRED_LOGIN_TOKEN

**Review by piotrjurkiewicz — APPROVED** [review]

File: .env.sample [hunk 5]
```diff
@@ -1,22 +1,30 @@
...
+# Source of binaries of app dependencies.
 CI_API_V4_URL=http://example.com
+# Credentials for binaries of app dependencies.
 NVPN_LINUX_GL_DEPS_CREDS=user:password
+# Domains for analytics.
 EVENTS_PROD_DOMAIN=http://example.com
 EVENTS_STAGING_DOMAIN=http://example.com
 EVENTS_SUBDOMAIN=subdomain
-FEATURES="telio drop"
+# App features, possible values are:
```
Comment (mariusSincovici) [inline .env.sample:22]: Shouldn't this also contain `firebase` value and that when it is defined `FIREBASE_TOKEN` is needed?

Comment (bartoszWojciechO) [reply .env.sample:22]: Afaik it was removed because it's not relevant to OSS. I'm working on a confluence page, it's included there.

**Review by mariusSincovici — APPROVED** [review]

**Pull request merged**